index.rst

```diff
@@ -36,13 +44,12 @@
 
    Feel free to delete this instructional comment.
 
-:tocdepth: 1
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-.. sectnum::
+.. abstract::
 
-   We rely on particular tag formats to order and present Lab images to users.  This Technote is intended to formalize and document those formats.
+   We rely on particular tag formats to order and present Lab images to users.
+   This Technote is intended to formalize and document those formats.
```

Comment:
Ideally the abstract would have a bit more context, since in isolation it's not obvious what a "Lab image" is. Maybe a sentence here with a brief definition of Nublado and then say this is the version numbering convention for the JupyterLab images for Nublado user pods.

Nit: We mostly use "technote" and not "Technote."